app/scoring/tests/test_scores_routes.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+import os.path
+import pytest
+from flask import url_for
+from app.common.static import get_dict_from_json_file
+
+JSON_DIRECTORY = (
+    "cypress/fixtures"  # TODO: move to suitable python directory and make robust
+)
+
+
+def is_uuid(string):
```

Comment:
I'm sure we have some function to check if uuid is valid or no, no need to create a new one

Reply:
> I'm sure we have some function to check if uuid is valid or no, no need to create a new one

We do have app.common.validate_uuid, but it raises one of our CustomErrors, which have HTTP status_codes, which are not relevant here. But perhaps that is too conservative and I should just go ahead and use it? In principle I agree with you.

Reply:
So basically you have the same logic `try-except + uuid.UUID(string)`. It's a code duplication.
I recommend you move your new function to the common folder and use it here and in `app.common.validate_uuid`.

Reply:
Agreed. Finally addressed this in today's commits.

I did notice something else, however, but it seems out of scope here and I suggest creating a separate issue: app.common.uuid.validate_uuid (which is used in many places) takes two arguments, but its checks (only) apply to the first one and it's error messages (only) refer to the second one.

Reply:
> I did notice something else, however, but it seems out of scope here and I suggest creating a separate issue: app.common.uuid.validate_uuid (which is used in many places) takes two arguments, but its checks (only) apply to the first one and it's error messages (only) refer to the second one.

Though I don't know, maybe this was intentional, as the second argument does make the error messages a little more precise ...